testsuite/features/step_definitions/common_steps.rb

```diff
@@ -729,6 +729,32 @@
   end
 end
 
+When(/^I disable SUSE Manager tools repositories on "([^"]*)"$/) do |host|
+  node = get_target(host)
+  os_version, os_family = get_os_version(node)
+  if os_family =~ /^opensuse/ || os_family =~ /^sles/
+    repos, _code = node.run('zypper lr | grep "tools" | cut -d"|" -f2')
+    node.run("zypper mr --disable #{repos.gsub(/\s/, ' ')}")
+  elsif os_family =~ /^centos/
+    repos, _code = node.run('yum repolist enabled 2>/dev/null | grep "tools" | cut -d" " -f1')
+    repos.gsub(/\s/, ' ').split.each do |repo|
+      node.run("sed -i 's/enabled=.*/enabled=0/g' /etc/yum.repos.d/#{repo}.repo")
+    end
+  end
+end
```

Comment:
I hope we never pass Sonarcube in our code, as we are adding a lot of duplicated code :S 
I would also refactor this code in the manner you wish, to don't have two "step definitions" doing exactly the same behavior but with the opposite result.

Reply:
Same, feel free to do it. I don't think it helps readability, and strictly speaking it slows down execution a bit, but ok.

Reply:
:laughing: Really do you feel the time of execution will change more than a microsecond?
I would prefer we refactor those steps to use both enable/disable options, but I will not go behind other's code refactoring it in another PR...